beamerthemeuulm.sty

```diff
@@ -198,131 +198,259 @@
 	\end{minipage}
 }
 
+% corresponds to '\halfpage', however, with the new columns we could use larger values
+% (same for thirdnext)
+\def\halfnext{.489\textwidth}
+% corresponds to '\thirdpage'
+\def\thirdnext{.31\textwidth}
+\def\uulm@cols@error{\PackageError{beamerthemeuulm}{The layout <TODO> you selected does not allow another \string\mynextcolumn\space (page: \thepage, frame: \insertframenumber,\on@line)}{Either remove the command, or choose another Layout}}
```

Comment:
Is this a warning or an error? I guess a warning would be enough here.

Reply:
Currently i made everything an error. But i can change that to warning.